### PR TITLE
ocrvs-3717: fixed document title name

### PR DESCRIPTION
**Pull request by CornelMaze**
- Added the document title after the componentDidMount

### CODE REVIEW

**Review by jpye-finch — COMMENTED** [review]
This Title should not be hardcoded. Should be taken from what is defined in application configuration

**Comment by CornelMaze** [comment]
> This Title should not be hardcoded. Should be taken from what is defined in application configuration

Ok

File: packages/login/src/views/resetCredentialsForm/forgottenItemForm.tsx [hunk 1]
```diff
@@ -58,7 +63,9 @@ class ForgottenItemComponent extends React.Component<Props, State> {
   }
 
   render() {
-    const { intl, goToHome } = this.props
+    const { intl, goToHome, applicationName } = this.props
+    if (applicationName) document.title = applicationName
```
Comment (Zangetsu101) [inline packages/login/src/views/resetCredentialsForm/forgottenItemForm.tsx:67]: @CornelMaze Although this should work, if you think about it the document title would be set each time the component re-renders which we do not want. We should go with a similar approach used in #4233.

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

File: packages/login/src/views/resetCredentialsForm/forgottenItemForm.tsx (hunk 1 above)
Comment (CornelMaze) [reply packages/login/src/views/resetCredentialsForm/forgottenItemForm.tsx:67]: @Zangetsu101 Yeah. That's true.

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**